src/Slides.js

```diff
@@ -0,0 +1,25 @@
+import { URL } from 'url';
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  if (!['slides.com', 'www.slides.com', 'team.slides.com'].includes(host)) {
+    return false;
+  }
+
+  const pathTrimmed = pathname.split('/').filter(Boolean);
+  return pathTrimmed.length === 2;
+};
+
+export const getSlidesIFrameSrc = urlString => {
+  const { host, pathname, hash } = new URL(urlString);
+  // remove trailing slash
+  const pathnameTrimmed = pathname.replace(/\/+$/, '');
+  return `https://${host.replace(/w{3}\./, '')}${pathnameTrimmed}/embed${hash}`;
```

Comment:
No need to use a regex, since the only thing that can be before `slides.com` is `www.` or `team.`

```suggestion

  return `https://${host.replace('www.', '')}${pathnameTrimmed}/embed${hash}`;
```